hedera-node/src/main/java/com/hedera/services/contracts/ContractsModule.java

```diff
@@ -69,7 +70,9 @@
 import static com.hedera.services.contracts.sources.AddressKeyedMapFactory.storageMapFrom;
 import static com.hedera.services.files.EntityExpiryMapFactory.entityExpiryMapFrom;
 
-@Module
+@Module(includes = {
```

Comment:
I actually thought it was necessary as CodeCache is now being injected in and this is where the class is kept. But now that you mention it, I do think it's a good signal to others that something from the store package is coming in. Otherwise searching for thing is very Spring Boot-like ... it could be anywhere and you have to go searching. What do you think?